components/jupyterhub/README.md

```diff
@@ -42,7 +32,14 @@ The above will expose JupyterHub on http://localhost:8000. The pod name can be o
 
 ## Configuration
 
-Configuration for JupyterHub is shipped separately and contained within the configmap under `manifests/config.yaml`. It is a Python file that is consumed by JupyterHub on starting up. The supplied configuration has reasonable defaults for the requisite fields and **no authenticator** configured by default.
+Configuration for JupyterHub is shipped separately and contained within the configmap defined by the [core componenent](https://github.com/google/kubeflow/tree/master/kubeflow). It is a Python file that is consumed by JupyterHub on starting up. The supplied configuration has reasonable defaults for the requisite fields and **no authenticator** configured by default. Furthermore, we provide a number of parameters that can be used to configure
+the core component. To see a list of ksonnet parameters run
+
+```
+ks prototype describe kubeflow-core
+```
+
+If the provided parameters don't provide the flexibility you need, you can take advantage of ksonnet to customize the core component and use a config file fully specified by you.
```

Comment:
Maybe specify how to get the manifest here?

Reply:
Filed #56